docs/reference/snapshot-restore/apis/create-snapshot-api.asciidoc

```diff
@@ -102,10 +102,25 @@ The cluster state includes:
 * Legacy index templates
 * Ingest pipelines
 * {ilm-init} lifecycle policies
+* Data stored in system indices, such as Watches and task records (configurable via `feature_states`)
```

Comment:
There isn't a great place to leave this comment but this is in the section about what the `cluster state` contains. IMO we should change the wording above to say global state instead of cluster state. Alternatively we can move this out of being included in the cluster state and say that global state also includes data stored in system indices.